Create physical_entity.md

cgendreau opened the pull request.

cgendreau committed:
Create physical_entity.md

cboelling commented:
So how is the process of reflecting e.g. today's discussion in this merge request - are additional changes made directly to physical_entity.md to accommodate them?

cgendreau committed:
Updated cardinalities

Based on discussions on 2020-09-30

cgendreau commented:
Yes, I just updated the document in this Pull Request

cgendreau committed:
Update physical_entity.md

fixed typo

falkogloeckler commented on component_descriptions/physical_entity.md:
```diff
@@ -0,0 +1,47 @@
+* Substrate type, air/water/soil
+* Potentially a History of State of the object - frozen/ dead/ in water/etc… (Link to preparation and protocol)
+* Amount? weight/ countable unit (3 parts of something) ( Talk to Falko for more)
+* Contaminated? Or another better term for this.
```
As of the discussion on 16th Sept. this should be one (of potentially many) values of the field status, right?

cgendreau replied:
You mean Contaminated should be a value of another field representing the status like "Potentially a History of State of the object" ?

falkogloeckler replied:
Yes

cgendreau replied:
Made some adjustments

falkogloeckler commented on component_descriptions/physical_entity.md:
```diff
@@ -0,0 +1,47 @@
+* Amount? weight/ countable unit (3 parts of something) ( Talk to Falko for more)
+* Contaminated? Or another better term for this.
+* Date status changed
+  * State = destroyed
```
This should not be binary
// todo: add more like "consumed", "lost", ...

cgendreau committed:
Merged the different text regarding the state

cgendreau committed:
Move history under state

Since the state can evolve: frozen -> contaminated -> destroyed the history is more important.

falkogloeckler reviewed (approved):
The values of states need some more discussion in order to align with preparation (for example: Does "frozen" mean the preparation type here or just the decontamination method?) 
But as the peparation component is still to be created, let's keep this jn mind for later.

cgendreau commented:
Indeed, I will change it so it doesn't look like a final list

cgendreau committed:
Trying to clarify state of the physical entity

The decontamination period would also be a state, just not sure how it would be called.
